packages/x-ui-admin/src/react/components/AdminListItemRenderer.tsx

```diff
@@ -121,7 +121,11 @@ export function AdminListItemRenderer(props: AdminListItemRendererProps) {
 
     value = (
       <Link to={props.relation.path}>
-        <Tag>{props.value[props.relation.dataIndex]}</Tag>
+        <Tag>
+          {props.relation.dataIndex
```

Comment:
you've used this in many places. Recommend you use: https://www.npmjs.com/package/lodash.get